libpod/events/journal_unsupported.go

```diff
@@ -0,0 +1,12 @@
+// +build !systemd
+
+package events
+
+import (
+	"fmt"
```

Comment:
Could you use errors for consistancy.

Reply:
Yes alright, I added a new error `libpod.ErrNoJournaldLogging`.